### PR TITLE
Add BattleOrders tardy check and quit/continue vars

### PR DESCRIPTION
**Pull request by jmichelsen**
This removes the unnecessary delay when someone rejoins a game or joins
late and the time for battle orders has passed.

The bot will now check for other player locations and just continue on instead of waiting the
BattleOrder timeout. Instead of quitting if the timeout is reached, now the character can optionally just continue on with the run. These two changes combined make for smoother battle order control.

### CODE REVIEW

File: d2bs/kolbot/libs/bots/BattleOrders.js [hunk 1]
```diff
@@ -1,12 +1,63 @@
 /**
 *	@filename	BattleOrders.js
-*	@author		kolton
+*	@author		kolton, jmichelsen
 *	@desc		give or receive Battle Orders buff
+*	@return {boolean}
```
Comment (jaenster) [inline d2bs/kolbot/libs/bots/BattleOrders.js]: can you remove this? =) If so we are good to go if you ask me

Comment (jmichelsen) [reply d2bs/kolbot/libs/bots/BattleOrders.js]: Thanks J :) 

Is there a negative to having this return type specified? I added it to satisfy the linter that complains that "Primitive value returned from constructor will be lost when called with 'new' "

Comment (jaenster) [reply d2bs/kolbot/libs/bots/BattleOrders.js]: You got the right linter file?🙃

Comment (jmichelsen) [reply d2bs/kolbot/libs/bots/BattleOrders.js]: Sorry, it's actually Webstorm that complains when you don't specify a return in the docstring on functions like this. Still, will it cause problems to leave it in? I can pull it out either way, if that's what we'd prefer. Just curious.

Comment (jmichelsen) [reply d2bs/kolbot/libs/bots/BattleOrders.js]: I removed this. It's ready for rereview or approval :)

**Pull request merged**